### PR TITLE
Heroku Proxy: allow selecting public keys

### PR DESCRIPTION
**Pull request by harley**
## Context

I have many SSH public keys under my GitHub account. Currently the tool selects the first key, which isn't the same key I have on my work laptop.

## Change

Augment the proxy request form to include a dropdown selecting which public key to use

- [x] Update README
- [x] Update `/heroku` page

### Before
(without the Sessions is creating screen, but you get the idea)

screen shot 2018-07-03 at 11 01 18 am

### After
screen shot 2018-07-03 at 12 06 50 am

## Consideration

I need to store `public_key_title` in `HerokuAccess`. I also store `public_key` to avoid another trip to GitHub every time we need the key. Let me know if this is a concern.

## Misc

- Updated to Ruby 2.5.1 because I didn't want to install the old 2.4.1 :)

@anvox

### CODE REVIEW

**Comment by anvox** [comment]
If stored public key, we'll have 2 source of truth. I'm afraid the case if key is invalidated on github but still on our system. But in case it come from user action, I think it's OK.

File: src/lib/github_authenticator.rb [hunk 1]
```diff
@@ -71,6 +71,15 @@ def public_key
                     end
     end
 
+    def verified_keys
+      @client.keys.select {|e| e[:verified]}.map{ |e| [e[:title], e[:key]] }.to_h
```
Comment (anvox) [inline src/lib/github_authenticator.rb]: Bad naming :P

**Comment by anvox** [comment]
@harley It's deployed with a small changes that `Dotenv.load` if possible. I dont want to use it on server. I think it's good for development, but bring unnecessary complexities to server run :) Thanks.

**Comment by harley** [comment]
Agree! Thanks. Can use Dotenv.load if defined?(Dotenv)

and only have dotenv gem in development group which isnt loaded when Rack
env is development

**Pull request merged**